Makefile

```diff
@@ -1,4 +1,4 @@
-DEV_ROCKS = busted luacheck lua-llthreads2
+DEV_ROCKS = busted "luacheck 0.20.0" lua-llthreads2
```

Comment:
While we're at it, shouldn't we also pin the versions for `busted` and `llthreads2`. History is waiting to repeat itself otherwise...